packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -87,11 +87,9 @@ function Element({
     baseColor,
   } = resource;
 
-  const { isCurrentResourceProcessing } = useLocalMedia(
-    ({ state: { isCurrentResourceProcessing } }) => ({
-      isCurrentResourceProcessing,
-    })
-  );
+  const { isCurrentResourceProcessing } = useLocalMedia(({ state }) => ({
+    isCurrentResourceProcessing: state.isCurrentResourceProcessing,
+  }));
```

Comment:
Why change this?

Reply:
Improved readability. Three lines instead of five.

Reply:
I made a point of changing all these to be the other style for readability. I liked it better the way it was before. We should pick one and stick with it.

Reply:
The other style makes sense when there are more things to return.

Not worth enforcing a specific style for everything IMO.